frontend/src/css/modules/base.scss

```diff
@@ -67,6 +67,23 @@ input.dark-autofill:-webkit-autofill:active {
     -webkit-text-fill-color: #4d4d4d !important;
 }
 
+input[type="radio"].selectTeam + label {
```

Comment:
`[type="radio"]:checked + label:after` But, this line can override any checked radio in future, How will you protect this?